sys/include/net/gnrc/udp.h

```diff
@@ -54,6 +54,13 @@ extern "C" {
 #endif
 
 /**
+ * @brief   Default UDP port range for dynamic source port allocation
+ */
+#define GNRC_UDP_PORTRANGE_MIN (49152U)
```

Comment:
you're right, there are lots of port ranges one can possibly define, but at the moment we only need this one :) Because, until now there was not a single portrange for UDP in RIOT ...

That said, I tend to agree that the names could be a little more specific (or speaking), how about `GNRC_UDP_SRC_PORTRANGE_*`? Because `EPHEMERAL` is not very helpful, I agree with @miri64 here.